test/corpus/string/multiLineInterpolation.txt

```diff
@@ -1,5 +1,5 @@
 ================================================================================
-multiline
+string/multilineInterpolation
```

Comment:
```suggestion
string/multiLineInterpolation
```

(or rename the file)

Reply:
The `string/` prefix is a little redundant, no? These tests are hierarchical and tree-sitter tells you which test this belongs in.

E.g. from their test report:

```
  basic:
    annotation:
        1. ✓ annotation
    comments:
        2. ✓ comments
    types:
        3. ✓ types
  class:
    constModifier:
        4. ✓ constModifier
    fixedModifier:
        5. ✓ fixedModifier
```

Reply:
Argh, I did this but of course GIt had trouble tracking this thanks to case insensitive file systems.